src/components/edit/EditTabsManager.tsx

```diff
@@ -0,0 +1,42 @@
+'use client';
+import React from 'react';
+import { useSearchParams, usePathname, useRouter } from 'next/navigation';
+
+import { Tabs, TabsList, TabsTrigger } from '@/components/ui/tabs';
+
+export default function EditTabsManager({
```

Comment:
Please use named exports instead of default to follow Coding conventions

Reply:
I changed an export default to named export.

Reply:
Thank you LGTM!